var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -44,6 +44,7 @@ class PyNumpy(Package):
     variant('lapack', default=True)
 
     extends('python')
+    depends_on('py-setuptools', type='build')
     depends_on('py-nose', type='build')
     depends_on('py-setuptools', type='build')
```

Comment:
Thanks @adamjstewart . Keen eye :-) I think that this must be a result of my rebaseing before committing. I'll fix this now.